src/common/k8s-api/kube-object.store.ts

```diff
@@ -167,21 +171,21 @@ export abstract class KubeObjectStore<T extends KubeObject> extends ItemStore<T>
     }
   }
 
-  protected async loadItems({ namespaces, api, reqInit, onLoadFailure }: KubeObjectStoreLoadingParams<T>): Promise<T[]> {
-    if (!this.context?.cluster.isAllowedResource(api.kind)) {
+  protected async loadItems({ namespaces, reqInit, onLoadFailure }: KubeObjectStoreLoadingParams): Promise<T[]> {
```

Comment:
Why was `api` passed in as an argument? Could it be different than `this.api`?

Reply:
It used to be possible for it to be different. When a store supported several apis. But that was removed. This makes it clearer and should remove a possible future bug.